lib/cisco_node_utils/radius_server.rb

```diff
@@ -39,6 +39,24 @@ def initialize(name, instantiate=true)
       end
       @name = name
 
+      if platform == :ios_xr
```

Comment:
It does this because in Nexus, we use separate setter methods whilst in EXR, we have to use the initialize.